bin/main.py

```diff
@@ -52,6 +53,15 @@ def run():
         else:
             Thread(target=process, args=(line, None)).start()
 
+def run_ipc():
+    receive_signal = PluginLoader('ipc_plugin.ipc_mod').receive_signal
```

Comment:
Maybe you should create a PluginLoader object (with a different name, see below) while parsing the arguments, and use its functions directly rather than assigning.